scripts/brp-remove-info-dir

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+# If using normal root, avoid changing anything.
+if [ -z "$RPM_BUILD_ROOT" -o "$RPM_BUILD_ROOT" = "/" ]; then
+	exit 0
+fi
+
+INFODIR=`rpm --eval %{_infodir}/dir`
+
+dir="$RPM_BUILD_ROOT/$INFODIR"
+
+if [ -f $dir -a ! -f "$RPM_BUILD_ROOT/sbin/install-info" -a ! -f "$RPM_BUILD_ROOT/usr/sbin/install-info" ]; then
```

Comment:
Some 'make install' rules uses 'install-info' to install info pages into install destination.
When installing info pages using 'install-info', it'll also register the info page in /usr/share/info/dir, for which in a clean buildroot doesn't exist and a new /usr/share/info/dir file with the info page entry gets created.

So this ensures that no /usr/share/info/dir ends up in package, conflicting with existing system /usr/share/info/dir